lib/alex_koin/slack.ex

```diff
@@ -68,6 +76,14 @@ defmodule AlexKoin.SlackRtm do
     end
   end
   defp create_reply(user, _, {:list_koins, _text}), do: SlackCommands.get_coins(user_wallet(user))
+  defp create_reply(user, _, {:help, _text}), do
+    message =
+      @available_commands
+        |> Enum.keys
+        |> Enum.join("  \n- ")
+
+    {"Available commands must include  \n", thread_ts: message_ts(message)}
```

Comment:
In this instance, you can use `message_ts: message_ts(message)` and it will figure out if it should be a thread or not.

Reply:
This just helps in case someone asks within a thread, we don't want to try to thread again.